ui/lib/src/elements/where-controller.ts

```diff
@@ -257,20 +217,9 @@ Folks:
 ${folks}
 </div>
 
-<mwc-dialog id="space-dialog" heading="Space" @closing=${
-      this.handleSpaceDialog
-    }>
-<mwc-textfield id="space-dialog-name" minlength="3" maxlength="64" placeholder="Name" required></mwc-textfield>
-<mwc-textfield id="space-dialog-url" placeholder="Image URL" required></mwc-textfield>
-<mwc-formfield label="Multi-wheres per user">
-<mwc-checkbox id="space-dialog-multi"></mwc-checkbox>
-</mwc-formfield>
-<mwc-button slot="primaryAction" dialogAction="ok">ok</mwc-button>
-<mwc-button slot="secondaryAction"  dialogAction="cancel">cancel</mwc-button>
-<img id="sfc" src=""></img>
-</mwc-dialog>
-
-<where-space id="where-space" .current=${this._current} avatar="${
+<where-space-dialog id="space-dialog" .current="${this._current}"></where-space-dialog>
+
+<where-space id="where-space" .current={$this._current} avatar="${
```

Comment:
Also careful with this.  If you don't put a `.` before the input, it won't be reactive.

So rather than `avatar="${this._myAvatar}"`, it's `.avatar=${this._myAvatar}`.

More docs: https://lit.dev/playground/#sample=examples/expressions and https://lit.dev/docs/templates/expressions/